staging/kos/cmd/controllers/subnet/validator/main.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"math/rand"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+	"time"
+
+	"github.com/golang/glog"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/workqueue"
+
+	kosclientset "k8s.io/examples/staging/kos/pkg/client/clientset/versioned"
+	kosinformers "k8s.io/examples/staging/kos/pkg/client/informers/externalversions"
+	"k8s.io/examples/staging/kos/pkg/controllers/subnet"
+	_ "k8s.io/examples/staging/kos/pkg/controllers/workqueue_prometheus"
+	_ "k8s.io/examples/staging/kos/pkg/util/parse/network/subnet/v1alpha1"
```

Comment:
This is out of context --- it is not evident from the content of this file why this import should be here.  It would be better if `import "k8s.io/examples/staging/kos/pkg/util/parse"` took care of this.

Reply:
Fixed in 02883e7.